src/app/api/v1/hibp/notify/route.ts

```diff
@@ -6,6 +6,7 @@ import { NextRequest, NextResponse } from "next/server";
 import { bearerToken } from "../../../utils/auth";
 
 import { PubSub } from "@google-cloud/pubsub";
+import { isFlagEnabled } from "../../../../functions/server/featureFlags";
```

Comment:
Just a thought, but we should have a place to keep track of all the feature flags (maybe README) so even for developers, we don't lose track of what we should set locally

Reply:
Ticket is https://mozilla-hub.atlassian.net/browse/MNTOR-2036 I've added that to acceptance criteria